tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -84,6 +84,13 @@ Given('the setting {string} of app {string} has been set to {string}', function
     ])
 })
 
+Given('app {string} has been enabled', function (app) {
```

Comment:
We need to disable the app after the test run. Please track and rollback this change in `After`.

Reply:
Its already done